include/msgpack/unpack_template.h

```diff
@@ -93,10 +92,9 @@ msgpack_unpack_func(msgpack_unpack_object, _data)(msgpack_unpack_struct(_context
     return (ctx)->stack[0].obj;
 }
 
-
 msgpack_unpack_func(int, _execute)(msgpack_unpack_struct(_context)* ctx, const char* data, size_t len, size_t* off)
 {
-    assert(len >= *off);
+    /* assert(len >= *off); */
```

Comment:
Commenting out the assertion changes a meaning of code in DEBUG mode.
Could you keep the assertion above like as follows?

```C++
    assert(len >= *off);
    {
        ...
```